src/__tests__/Question.js

```diff
@@ -149,3 +149,39 @@ it('should set incorrect answer to red and correct answer to green on submit', (
     ).toMatch(/color:green/);
   });
 });
+
+//Show Explanation Button should pop up when users submit an answer
```

Comment:
This looks good. Congrats on your first unit tests with enzyme!

Though it can be difficult, ideally your tests are not brittle and can stand the test of time. This means that they don't test implementation details. For our case, that might mean not checking to make sure an element is rendered by looking at the class name, not verify the text of a question or answer, etc. I don't think you need those promises, but I don't know for sure.

I'm all for merging these tests. Thanks!

Reply:
I normally put a render update in a promise but it doesn't seem to be necessary here having tested.